apps/console/src/extensions/configs/models/documentation.ts

```diff
@@ -0,0 +1 @@
+export type DocumentationLinksExtensionInterface = Record<string, unknown>
```

Comment:
Missing license header.
```suggestion
export type DocumentationLinksExtensionInterface = Record<string, unknown>;
```

Reply:
Fixed with 642a464ebd9d74c5bde502d428ed8c8d82e0e211